easybuild/easyconfigs/w/WRF/WRF-4.5.1-foss-2023a-dmpar.eb

```
@@ -0,0 +1,53 @@
+name = 'WRF'
+version = '4.5.1'
+buildtype = 'dmpar'
+versionsuffix = '-%s' % buildtype
+
+homepage = 'https://www.wrf-model.org'
+description = """The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale
+ numerical weather prediction system designed to serve both operational forecasting and atmospheric
+ research needs."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'opt': False}  # don't use agressive optimization, stick to -O2
+
+source_urls = ['https://github.com/wrf-model/WRF/archive/']
+# sources = ['v%(version)s.tar.gz']
+sources = [{
+    'filename': 'v%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/wrf-model',
+        'repo_name': 'WRF',
+        'tag': 'v%(version)s',
+        'clone_into': 'WRF-%(version)s',
+        'recursive': True,
+    },
+}]
+patches = [
+    'WRF-4.5.1_netCDF-Fortran_separate_path.patch',
+]
+checksums = [
+    '9d557c34c105db4d41e727843ecb19199233c7cf82c5369b34a2ce8efe65e2d1',  # v4.5.1.tar.gz
```

can't specify a checksum since source tarball is created on the fly from cloned Git repo
```suggestion
    None,  # v4.5.1.tar.gz
```